### PR TITLE
Bucket / customizer dqa adjustments

### PR DESCRIPTION
**Pull request by alliuca**
**Description:** Merges the Customizer DQA adjustments bucket branch:
- Fix background color around tickets / rsvp blocks (remove extra white)
- Add body class to target AR pages with shortcode

**Ticket:** https://moderntribe.atlassian.net/browse/TEC-3699

**Screenshots:**

Screen Shot 2020-12-22 at 3 36 16 PM

### CODE REVIEW

File: src/Tribe/Attendee_Registration/Template.php [hunk 1]
```diff
@@ -104,6 +104,17 @@ public function is_on_ar_page() {
 		return tribe( 'tickets.attendee_registration' )->is_on_page();
 	}
 
+	/**
+	 * Convenience wrapper for tribe( 'tickets.attendee_registration' )->is_using_shortcode() usage.
+	 *
+	 * @since TBD
+	 *
+	 * @return boolean
```
Comment (sc0ttkclark) [inline src/Tribe/Attendee_Registration/Template.php:112]:
```suggestion
	 * @return boolean Whether the Attendee Registration shortcode is being used.
```

**Review by sc0ttkclark — CHANGES_REQUESTED** [review]
Just a simple but important tweak

**Review by paulmskim — COMMENTED** [review]
looks good, just the one tweak suggested and common update

File: src/Tribe/Attendee_Registration/Template.php (hunk 1 above)
Comment (alliuca) [reply src/Tribe/Attendee_Registration/Template.php:112]: thanks! fixed that in https://github.com/the-events-calendar/event-tickets/pull/1919

**Review by bordoni — APPROVED** [review]

**Review by bordoni — APPROVED** [review]

**Review by bordoni — APPROVED** [review]

**Pull request merged**